content/en/blog/_posts/2021-04-23-metrics-stability-ga/2021-04-23-metrics-stability-hits-ga.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: blog
+title: 'Kubernetes 1.21: Metrics Stability hits GA'
+date: 2021-04-23
```

Comment:
Will this not go live until Friday then?

Reply:
Correct, theres a cronjob that runs twice daily (IIRC) that will rebuild the site each day. Hugo won't build future dated blog posts \o/ Makes it a lot easier than having to manage holds like we used to.

Reply:
Great!! Looking forward to it